cloudbuild/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# This Dockerfile builds the image used in Cloud Build CI to run 'make test'.
+# To push a new image, run 'gcloud builds submit --project=cloud-eng-council --tag gcr.io/cloud-eng-council/make .'
```

Comment:
We should probably just put in place a trigger to do this automatically on master.

Reply:
Is it possible to do this trigger only when the related files are touched on master?

Reply:
Yes, you can configure cloud build triggers to trigger based on path.

Reply:
got it. I added a target to the makefile to build the new image when this new trigger runs

Reply:
We should have a cloudbuild for this as well, no? Go ahead and configure the trigger if you can.